api/lib/src/lib.rs

```diff
@@ -373,6 +374,7 @@ pub trait BrokerApi: SignedExtrinsicApi + Sized + Send + Sync + 'static {
 					channel_metadata,
 					boost_fee: boost_fee.unwrap_or_default(),
 					affiliate_fees,
+					refund_parameters,
```

Comment:
im sorry, i just only noticed this - can we add this information to the `SwapDepositAddressReady` event? otherwise we wont be able to detect if a new channel has refund params on the product side

Reply:
does not need to be in this pr, but would be great if we could add it soonish